doc/production-solr-collection-swap-process.md

```diff
@@ -10,16 +10,18 @@ These steps are broken down into three sections as these parts can be run sequen
 1. Initiate Alma FTP Export and wait for completion.
 1. Trigger the `catalog_move_alma_sftp_to_s3` Dag and wait for completion.
    1. Skip this step if you want to reuse the s3 alma_sftp files from the last run.
-3. If needed, move the latest boundwith export file from almasftp to s3 (To be automated).
+1. Trigger the `catalog_move_alma_sftp_bw_to_s3` Dag and wait for completion.
+   1. Skip this step if you want to reuse the s3 alma_sftp files from the last run.
```

Comment:
@ebtoner this is the bit that mentions new process.  But I just noticed I forgot to remove the old doc will push up a quick fix to that.